templates/tfengine/components/iam_members/terraform.tfvars

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{range $resource, $members := get . "iam_members"}}
+  {{$resource}}_iam_members = [
+    {{range $index, $element := $members}}
```

Comment:
try adding a - at the end of at the front so it can remove extra lines in the generated config.

e.g. {{range $index, $element := $members -}}

I'm not sure which one will work without playing with it. Could you try?